pkgs/tools/package-management/librepo/default.nix

```diff
@@ -35,11 +35,10 @@ stdenv.mkDerivation rec {
     libxml2
     glib
     openssl
-    zchunk
     curl
     check
     gpgme
-  ];
+  ] ++ stdenv.lib.optional stdenv.isLinux zchunk;
```

Comment:
Can you add a comment here summarizing zchunk currently is broken on darwin, but fine otherwise?

Reply:
Added a comment. I guess I could add one futher down as well, but the recipe is small enough that this is probably fine